src/main/java/it/gov/pagopa/pu/debtpositions/repository/DebtPositionRepository.java

```diff
@@ -50,4 +50,5 @@ WHERE EXISTS (
   @EntityGraph(value = "completeDebtPosition")
   DebtPosition findByInstallmentId(@Param("installmentId") Long installmentId);
 
+  DebtPosition findByIupdOrg(@Param("iupdOrg") String iupdOrg);
```

Comment:
is this annotation required?
```suggestion
  DebtPosition findByIupdOrg(String iupdOrg);
```